app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PublicRecipesController < ApplicationController
+  def index
```

Comment:
- [ ] OPTIONAL
Clicking on the public toggle button should either show or hide item in the public view. I think this will be a nice feature you can consider adding to your project in the future.

![image](https://user-images.githubusercontent.com/12745474/207640443-543077ee-f8d8-4370-80bd-6686623074a4.png)

Reply:
Will look on that. Thanks